build.gradle.kts

```diff
@@ -112,6 +112,19 @@ tasks.named<DependencyUpdatesTask>("dependencyUpdates").configure {
     }
 }
 
+// Gradle's "dependencies" task selector only executes on a single / the current project [1]. However, sometimes viewing
+// all dependencies at once is beneficial, e.g. for debugging version conflict resolution.
+// [1]: https://docs.gradle.org/current/userguide/viewing_debugging_dependencies.html#sec:listing_dependencies
+tasks.register("allDependencies") {
```

Comment:
Doesn't this need a description so that it appears in ./gradlew tasks without --all?